main.tf

```diff
@@ -48,14 +52,14 @@ resource "aws_s3_bucket" "pardot_form_handler_lambda" {
 data "archive_file" "lambda_source" {
   type = "zip"
 
-  source_dir  = "${path.module}/lambda-source"
-  output_path = "${path.module}/lambda-source/lambda-source.zip"
+  source_dir  = "${path.module}/dist"
+  output_path = "${path.module}/dist/${local.zip_name}"
```

Comment:
so after some testing, we should do away with this change and the added npm option. I forgot that the terraform script actually does the zipping. So in my tests, i was zipping a zip and thats why the handler was blowing up

Reply:
🤦🏽 That makes sense. I'll clean it up.